klippy/extras/replicape.py

```diff
@@ -205,6 +207,15 @@ def __init__(self, config):
         self.sr_spi = bus.MCU_SPI(self.host_mcu, sr_spi_bus, None, 0, 50000000)
         self.sr_spi.setup_shutdown_msg(self.sr_disabled)
         self.sr_spi.spi_send(self.sr_disabled)
+        if not self.host_mcu.is_fileoutput():
+            #pwm = [x for x in os.listdir('/sys/bus/platform/devices/48302200.pwm/pwm/')
+            #        if x.startswith('pwmchip')]
```

Comment:
I'd drop these commented out lines.

Reply:
yes, I forgot to remove them and they broke the CI due to line length